### PR TITLE
doc: describe how to deploy to Cloud Run

### LINKED ISSUE #50: Implement example deploying to Cloud Run

### PR DESCRIPTION
**Pull request by coryan**
Fixes #50

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -5,3 +5,90 @@ This directory contains a series of examples showing how to use the Functions Fr
 A good place to start is [hello_world](hello_world). It contains a simple example, with a simple function in a single
 `.cc` file.
 
+## Creating a Docker image for the examples
+
+The examples can run in Docker containers. To compile them you first need to
+[create a development image](#create-the-development-image). Once that image is available with this image at hand you
```
Comment (devbww) [inline examples/README.md:11]: Should _"Create the Development Image"_ just come first then?

File: examples/README.md [hunk 2]
```diff
@@ -5,3 +5,90 @@ This directory contains a series of examples showing how to use the Functions Fr
...
+
+```sh
+docker build -t gcf-cpp-develop -f build_scripts/Dockerfile build_scripts
+```
+
+## Creating a Docker image for the examples
+
+The examples can run in Docker containers. To compile them you first need to
+[create a development image](#create-the-development-image). Once that image is available with this image at hand you
+can create a docker image for any of the examples using:
```
Comment (devbww) [inline examples/README.md:21]: The three "image" references confuse me.

Perhaps s/with this image as hand//.

Or perhaps _"To compile them you first ... Once that image is available ... you can"_ now simplifies to _"Using the development image from above, you can"_.

**Comment by coryan** [comment]
PTAL

**Review by devbww — APPROVED** [review]

File: examples/README.md (hunk 2 above)
Comment (coryan) [reply examples/README.md]: Fixed, I think.

**Pull request merged**